genetico_tarea.py

```diff
@@ -105,7 +121,22 @@ def cruza_individual(self, cadena1, cadena2):
         #
         # ------ IMPLEMENTA AQUI TU CÓDIGO ----------------------------------
         #
-        raise NotImplementedError("¡Este metodo debe ser implementado!")
+        #raise NotImplementedError("¡Este metodo debe ser implementado!")
+        child = []
+        aux = list(range(0,len(cadena1)))
+        for _ in range(len(cadena1)):
```

Comment:
Cuando usas la variable en el for, no se utiliza _ como nombre de la variable, eso solo se usa cuando la variable no se utiliza explícitamente en el código, solo es una cuestión de estilo.